apis/v1alpha1/httproute_types.go

```diff
@@ -168,32 +183,65 @@ type HTTPRouteMatch struct {
 	ExtensionRef *RouteMatchExtensionObjectReference `json:"extensionRef" protobuf:"bytes,5,opt,name=extensionRef"`
 }
 
-// HTTPRouteFilter defines a filter-like action to be applied to
-// requests.
-type HTTPRouteFilter struct {
-	// Headers related filters.
-	//
-	// Support: extended
+const (
+	// FilterHTTPReqeustHeaderTransformer can be used to add or remove an HTTP
+	// header from an HTTP request before it is sent to the upstream target.
+	// Support: core
 	// +optional
-	Headers *HTTPHeaderFilter `json:"headers" protobuf:"bytes,1,opt,name=headers"`
+	FilterHTTPReqeustHeaderTransformer = "HTTPRequestHeaderTransformer"
```

Comment:
nit: Do we need the "Filter" prefix? This name is very long.

Reply:
The reason for adding `Filter` prefix is to easily determine what constants are filters.
This could potentially help with auto-completion as well.